daemon/authorization/crawler_get_new_release.py

```diff
@@ -0,0 +1,35 @@
+import configparser
+from spotify import Spotify
+from get_new_release import Get_new_release
+from get_album_song import Get_album_song
+
+# Read config.cfg to get Client_ID and Client_Secret
+config = configparser.ConfigParser()
+config.read('../config.cfg')
+Client_ID = config.get('Spotify_Parameters' ,'Client_ID')
+Client_Secret = config.get('Spotify_Parameters','Client_Secret' )
+
+# Get access token value
+s = Spotify(Client_ID,Client_Secret)
+access_token = s.get_token()
+
+
+new_release = Get_new_release(access_token)
+album_song = Get_album_song(access_token)
+
+# Get new release name. 
+# You will get twenty times as number
+new_count = 1
+album_information_list = []
+new_list = new_release.get_new_list(new_count)
```

Comment:
Maybe you must comment down what the structure is in each element on `new_list`.
It may cause some confusion.